docs/pingaccessRedirectless.md

```diff
@@ -0,0 +1,35 @@
+# PingAccess Redirectless support
+
+PingAccess specific callback functions used to build a configuration object for [redirectless mode](/docs/redirectless.md).    
```

Comment:
Perhaps this documentation could be expanded to mention what a developer would be able to do after completing the Redirectless flow?

Specifically, they can now access APIs protected by PingAccess

Reply:
How about:

> After successfully completing the redirectless flow, APIs protected by PingAccess can be accessed by configuring the fetch API or XMLHttpRequest API to include credentials. This configuration will ensure that the PingAccess session cookie, established by the redirectless flow, is included in API requests.